docs/spec/resolution.md

```diff
@@ -84,7 +84,7 @@
                 1. Set the _Next Update Commitment_ value to `null`.
                 2. Exit [update operation processing](#update-operation-processing) and advance to [compiled state processing](#compiled-state-processing).
```

Comment:
Note: we still have an open issue on update processing errors. https://github.com/decentralized-identity/sidetree/issues/834